packages/frontend/src/pages/single-play/components/SinglePlayResult.tsx

```
@@ -0,0 +1,200 @@
+        <div className="flex flex-col gap-2 overflow-hidden border-4 border-purple-400 bg-gradient-to-r from-slate-800/95 to-slate-900/95 p-4 shadow-2xl shadow-purple-500/30">
+          <h2
+            className="flex items-center gap-2 text-base font-bold text-purple-300"
+            style={{ fontFamily: 'Orbitron' }}
+          >
+            <i className="ri-file-list-3-line" />
+            QUESTION HISTORY
+          </h2>
+          {/* Scroll Area */}
+          <div className="flex flex-col items-stretch justify-start gap-2 overflow-y-auto bg-gradient-to-r from-slate-800/95 to-slate-900/95 pr-4 shadow-purple-500/30">
+            {questions.map((q, index) => (
+              <div className="flex flex-col items-stretch justify-center gap-2 border-2 border-slate-600 bg-slate-700/50 p-2">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)